### DISCUSSION TITLE
Censoring parts of URLs?

### DISCUSSION THREAD

**Discussion by scratchyone** [plausible/analytics#942] [Anything Else]
Is it possible to track certain URLs but censor parts of them? For example, I run a pastebin tool that gives each user a unique link to their uploaded paste. I want to track that a user viewed a paste, but I don't want the private URL of each paste showing up in my analytics dashboard.

**Comment by metmarkosaric** [comment]
That's not possible unfortunately. It has been requested before and there's this thread about it https://github.com/plausible/analytics/discussions/136
